### ISSUE TITLE
await import chrome.runtime.getURL causes UNSAFE_VAR_ASSIGNMENT

### ISSUE THREAD

**Issue by karlicoss** [mozilla/addons-linter#4686]
### Describe the problem and steps to reproduce it:

I'm trying to import `script.js` from a content script (`test.js`). `script.js` is declared as `web_accessible_resources` in my manifest, so this works perfectly in runtime.

```
// test.js
async function test () {
    await import (chrome.runtime.getURL('script.js'))
}
```

### What happened?

During running `web-ext lint`, I'm getting:

```
UNSAFE_VAR_ASSIGNMENT      Unsafe call to import for    Due to both security and performance concerns, this    test.js         2      11    
                           argument 0                   may not be set using dynamic values which have not                                  
                                                        been adequately sanitized. This can lead to security                                
                                                        issues or fairly serious performance degradation.      
```

### What did you expect to happen?

In this case the script is provided by the extension itself (`chrome.runtime.getURL('script.js')`), so this feels like a false positive?

In case I'm wrong about it being a false positive:

- what's wrong with it? Perhaps the error message could be improved in this case
- what's the right way to achieve the same effect? I could use `browser.tabs.executeScript({file: 'script.js'})` before injecting `test.js`, but that way the logic is kinda scattered away from the call site. 

### Anything else we should know?

Perhaps the closest relevant issue is https://github.com/mozilla/addons-linter/issues/2498, but it's different -- in that case it was a syntax error.

**Comment by Rob--W** [comment]
> In this case the script is provided by the extension itself (`chrome.runtime.getURL('script.js')`), so this feels like a false positive?

If the method is not overridden, then it would be safe indeed. But due to JavaScript's dynamic nature, it is not clear whether it is safe. For example, the code could be insecure if the snippet is part of a snippet that has a custom `chrome` variable, or if anything overrides the `chrome.runtime.getURL` method. Thus, the general warning is correct.

But Firefox does reject values other than extension scripts. In extension pages, the CSP should be blocking the import (just like any other way of loading scripts). In content scripts, here is a unit test that confirms that remote URLs are rejected: https://searchfox.org/mozilla-central/rev/28c0d45a553fd2817ac14e1562435e86dc0aa403/toolkit/components/extensions/test/xpcshell/test_ext_contentscript_module_import.js#28-70
